src/sentry/static/sentry/app/components/subscribeButton.jsx

```diff
@@ -0,0 +1,41 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'react-emotion';
+
+import Button from 'app/components/button';
+
+import {t} from 'app/locale';
+
+export default class SubscribeButton extends React.Component {
+  static propTypes = {
+    isSubscribed: PropTypes.bool.isRequired,
+    onClick: PropTypes.func.isRequired,
+  };
+
+  render() {
+    const {isSubscribed, onClick} = this.props;
```

Comment:
IMO rest the props here (`, ...props}`) so we can override things about the button should we wish on usage (size of button).

Reply:
Can we just add if needed? I don't see a use case right now since the style for the new incidents mockups is the same as the old subscribe button

Reply:
That's fine, I just try and make components like this (`xyzButton`) still accept the same props as a normal button